Source/MonoGame.Extended/Graphics/PrimitiveTypeExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace MonoGame.Extended.Graphics
+{
+    public static class PrimitiveTypeExtensions
+    {
+        public static int GetPrimitiveCount(this PrimitiveType primitiveType, int verticesOrIndicesCount)
+        {
+            switch (primitiveType)
+            {
+                case PrimitiveType.LineStrip:
+                {
```

Comment:
Here they can be removed no problem. It's just when each case statement has a fair bit of code that it's easier, in my opinion, to read. The curly braces also create a new stack frame which means I can use the same variable names inside each case statement without clashing. Again, really only useful when there is a fair bit of code per case statement.